README.md

```diff
@@ -36,7 +36,7 @@ Yes please!
 # Known Issues (maybe solutions...)
 
 ### Git not found (even though you installed it)
-Zigi needs to find the git executable in the 'PATH'. To determine the 'PATH' zigi sources /etc/proifile and .profile.
+Zigi needs to find the git executable in the 'PATH'. To determine the 'PATH' zigi sources /etc/proifile and ~.profile.
```

Comment:
Did you mean /etc/profile  (and not /etc/proifile) ?

Reply:
There are multiple places to define the path - there is the generic /etc/profile and then there is the users profile found in their home directory which is referenced by the ~.  There is also a bash way that I don't recall :)

Reply:
I think you have a typo in the /etc/profile,   The text says /etc/proifile (extra i)

Reply:
I believe you are correct - I will fix this.